lib/loader/loader.c

```diff
@@ -130,6 +130,16 @@ evmc_create_fn evmc_load(const char* filename, enum evmc_loader_error_code* erro
     return create_fn;
 }
 
+const char* evmc_cannot_open_error()
```

Comment:
I'd say this should be named `evmc_last_error_msg()` and document that the returned pointer should never be freed.

This also gives the flexibility to use a static buffer if Windows would return a heap pointer.